.github/workflows/tests.yml

```diff
@@ -242,11 +244,9 @@ jobs:
           call conda config --set always_yes yes
           call conda config --set auto_update_conda no
           call conda info
-          python -c "import sys; print(sys.version)"
-          python -c "import sys; print(sys.executable)"
-          python -c "import sys; print(sys.prefix)"
+          python -c "import sys; print(sys.version); print(sys.executable); print(sys.prefix)"
           call conda update -q --all||exit 1
-          call conda install -q pip python-libarchive-c pytest git pytest-cov jinja2 m2-patch flake8 mock requests contextlib2 chardet glob2 perl pyflakes pycrypto posix m2-git anaconda-client numpy beautifulsoup4 pytest-xdist pytest-mock filelock pkginfo psutil pytz tqdm conda-package-handling||exit 1
+          call conda install -q pip pytest git pytest-cov jinja2 m2-patch flake8 mock requests contextlib2 chardet glob2 perl pyflakes pycrypto posix m2-git anaconda-client numpy beautifulsoup4 pytest-xdist pytest-mock filelock pkginfo psutil pytz tqdm conda-package-handling conda-index||exit 1
```

Comment:
It would be nice if we could pull these out of the recipe or setuptools dependencies.